confgenerator/confgenerator.go

```diff
@@ -249,13 +249,13 @@ func defaultTails(logsDir string, stateDir string) (tails []*conf.Tail) {
 	return []*conf.Tail{
 		{
 			Tag:  "ops-agent-fluent-bit",
-			DB:   fmt.Sprintf("%s/buffers/ops-agent-fluent-bit", stateDir),
-			Path: fmt.Sprintf("%s/logging-module.log", logsDir),
+			DB:   filepath.Join(stateDir, "buffers", "ops-agent-fluent-bit"),
```

Comment:
It's fine to continue having "buffers/ops-agent-fluent-bit" here.

But, why is this needed at all if we're going to pass it in on the command line when starting fluentbit?

Reply:
it will have it in a not pretty syntax.
C:\ProgramData\.../buffers/ops-agent-fluent-bit

Reply:
Also for the question why it's needed here, because the configuration of fluent-bit needs to know where to tail the agent's own logs.